build/download-manifests/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/projectsyn/steward/pkg/images"
+)
+
+var crds = []string{
+	"https://raw.githubusercontent.com/argoproj/argo-cd/{{VERSION}}/manifests/crds/application-crd.yaml",
+	"https://raw.githubusercontent.com/argoproj/argo-cd/{{VERSION}}/manifests/crds/appproject-crd.yaml",
+}
+
+func main() {
+	path, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+
+	version, err := version(images.DefaultArgoCDImage)
+	if err != nil {
+		panic(err)
+	}
+
+	for _, urlTmpl := range crds {
+		err := download(strings.Replace(urlTmpl, "{{VERSION}}", version, -1), path+"/manifests/")
```

Comment:
Any reason to use `Replace` and `{{VERSION}}` instead of `Sprintf` and `%s`?  I mean this works no problem, but I was a little confused at first.